css/guide.css

```diff
@@ -160,6 +160,9 @@ ol.warning{
 	font-weight: bold;
 	border-bottom: none;
 }
+.itemGlitch{
```

Comment:
just add this to the .glitch section (and split off from the a.screenshot if needed) instead of creating a new .itemGlitch

Reply:
some inheritance thing is preventing .glitch on its own from working, but I realized I can do .item.glitch to select it more specifically to override it.